main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -292,6 +308,11 @@ class LanguageServer(port: Int) extends WebSocketServer(new InetSocketAddress("l
       }
     }
 
+    // Add JARs.
+    for (uri <- jars) {
+      flix.addJar(uri)
```

Comment:
I think there could be other places where this is need too? Can you check where else `addInput` is called in `LanguageServer`? Or where `sources` is used?

Reply:
I believe I got all the LSP places. ButI have to check `Shell` again

Reply:
OK, let me know when done, and then I will merge.

Reply:
Interactive mode now supports jars.